src/mParticleJSKitBoilerplate.js

```diff
@@ -209,10 +54,10 @@ var UserAttributeHandler = require('../integration-builder/user-attribute-handle
         }
 
         this.init = initForwarder;
-        this.process = processEvent;
-        this.setUserAttribute = setUserAttribute;
-        this.removeUserAttribute = removeUserAttribute;
         this.onUserIdentified = onUserIdentified;
+        this.process = function() {
```

Comment:
is this needed?

Reply:
this is needed because https://github.com/mParticle/mparticle-sdk-javascript/blob/master-v2/src/forwarders.js#L280

it doesn't do a check for `if forwarder[i].process`, it just runs it. events are still sent, they just disappear into this function now instead of erroring out. 

i can note to make a change in the core SDK to look for a `forwarder[i].process` and to only run it if it exists but for now we can keep this as is

Reply:
cool 👍